lib/validate-formation.js

```diff
@@ -0,0 +1,10 @@
+const joi = require('joi');
+
+module.exports = function (formation) {
+  joi.assert(formation, joi.array().items(
+    joi.object().keys({
+      type: joi.string().error(val => `need string for type got ${val}`),
+      quantity: joi.number()
+    }).options({ stripUnknown: true })
```

Comment:
`allowUnknown`? If this is actually stripping out the size is there a test case covering that...

Reply:
maybe add a test for `validate-formation`?

Reply:
updated to `allowUnknown: true`, and added test